src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -252,6 +251,6 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager'));
+        $this->app->set('assetManager', new Maybe());
```

Comment:
Alternative fix
```
$this->app->set('assetManager', Stub::make('yii\web\AssetManager', ['bundles' => false]));
```
then assetManager will consider all bundles as dummy and they won't be published. However developers can directly use
```
Yii::$app->assetManager->publish($path, $options);
```
and requested file or directory will be published and published incorrectly.
I can't say how widely direct call of `publish` is used, I never used it.